integration/app/src/main/cpp/lib.cpp

```diff
@@ -0,0 +1,9 @@
+#include <string>
```

Comment:
Remove this include since std::string is not used?

Reply:
lol, good call.  I only have this here to force an ABI split, so I didn't pay much attention to the quality of the code here.